scripts/prepare_sector_network.py

```diff
@@ -1874,10 +1874,12 @@ def add_biomass(n, costs):
         bus0=spatial.gas.biogas,
         bus1=spatial.gas.nodes,
         bus2="co2 atmosphere",
+        bus3="co2 stored",
         carrier="biogas to gas",
         capital_cost=costs.loc["biogas upgrading", "fixed"],
         marginal_cost=costs.loc["biogas upgrading", "VOM"],
         efficiency2=-costs.at['gas', 'CO2 intensity'],
+        efficiency3=costs.at["biogas", "CO2 stored"],
```

Comment:
Shouldn't be the same amount of CO2 stored which is reduced from the atmosphere?
so `-efficiency2=efficiency3=costs.at['gas', 'CO2 intensity']`